### PR TITLE
Jms intitial code review

### LINKED ISSUE #12: Review Activity class
* Review activity file

### PR DESCRIPTION
**Pull request by jordy1611**
### What’s this PR do?
* Adds comments to each method in Activity class constructor
* Adds comments to some of test file for activity class
* Comment out unused methods in Activity class
* Use comments to designate section of tests to rewrite using more dynamics methods created later activity class
### Where should the reviewer start?
* Activity.js
* activity-test.js
### How should this be manually tested?
* Just load the page and make sure it still works
### What are the relevant tickets?
resolves #12

### CODE REVIEW

File: src/Activity.js [hunk 1]
```diff
@@ -1,48 +1,59 @@
...
+  //     return true;
+  //   }
+  //   return false
+  // }
+  // getDaysGoalExceeded(id, userRepo) { // returns array of dates user exceeding goal steps
+  //   return this.activityData.filter(data => id === data.userID && data.numSteps > userRepo.dailyStepGoal).map(data => data.date);
+  // }
+  // getStairRecord(id) { // returns highest number of stairs user as ever done on single day
+  //   return this.activityData.filter(data => id === data.userID).reduce((acc, elem) => (elem.flightsOfStairs > acc) ? elem.flightsOfStairs : acc, 0);
+  // }
```
Comment (carlymclift) [inline src/Activity.js:40]: Glad you found all the methods that are not being used

File: src/Activity.js [hunk 2]
```diff
@@ -65,24 +76,25 @@ class Activity {
...
-  getStreak(userRepo, id, relevantData) {
-    let data = this.activityData;
+
+  getStreak(userRepo, id, dataType) { //return 3-day increasing streak for a users step count or active minutes
+    let data = this.activityData; // this is redundant
     let sortedUserArray = (userRepo.makeSortedUserArray(id, data)).reverse();
     let streaks = sortedUserArray.filter(function(element, index) {
       if (index >= 2) {
-        return (sortedUserArray[index - 2][relevantData] < sortedUserArray[index - 1][relevantData] && sortedUserArray[index - 1][relevantData] < sortedUserArray[index][relevantData])
+        return (sortedUserArray[index - 2][dataType] < sortedUserArray[index - 1][dataType] && sortedUserArray[index - 1][dataType] < sortedUserArray[index][dataType])
```
Comment (carlymclift) [inline src/Activity.js:90]: Much better naming for the data being used.

File: test/activity-test.js [hunk 3]
```diff
@@ -13,7 +14,7 @@ describe('Activity', function() {
   let userRepo;
   let activity;
 
-  beforeEach(function() {
+  beforeEach(function() { // create smaller sample, probably use before
```
Comment (carlymclift) [inline test/activity-test.js:17]: Definitely use the sample data set instead of this hard coded data.

File: src/Activity.js (hunk 2 above)
Comment (relyt4me) [reply src/Activity.js:90]: I agree I think that we need to make these as consistant as possible

File: test/activity-test.js (hunk 3 above)
Comment (jordy1611) [reply test/activity-test.js:17]: Yep! Just needed that file merged to start using it

File: src/Activity.js [hunk 4]
```diff
@@ -1,48 +1,59 @@
...
+  //   }
+  //   return false
+  // }
+  // getDaysGoalExceeded(id, userRepo) { // returns array of dates user exceeding goal steps
+  //   return this.activityData.filter(data => id === data.userID && data.numSteps > userRepo.dailyStepGoal).map(data => data.date);
+  // }
+  // getStairRecord(id) { // returns highest number of stairs user as ever done on single day
+  //   return this.activityData.filter(data => id === data.userID).reduce((acc, elem) => (elem.flightsOfStairs > acc) ? elem.flightsOfStairs : acc, 0);
+  // }
+  //// end not needed methods
```
Comment (relyt4me) [inline src/Activity.js:41]: Some of these might need to be used eventually when we get our activity section more user friendly

**Pull request merged**